CONTRIBUTING.md

```diff
@@ -1,143 +1,177 @@
+
+
+- [x] **Sync with Central Repository:** :arrows_counterclockwise: Ensure your fork is current with the central repository, facilitating a smooth merge.
+- [x] **Lint:** :wrench: Ensure that linters have run over all new or modified files
+- [x] **Spelling:** :abc: Ensure that linters have run over all new or modified files
+- [x] **Significant Changes:** :mag_right: Confirm that every modified file contributes meaningful content, steering clear of inconsequential changes like mere whitespace adjustments.
+- [x] **Testing:** :test_tube: Write tests for any new features, and update existing ones if you've made changes to functionalities.
+- [x] **Language & Spelling:** :book: Use English for all variable names, class names, functions, comments, etc., and ensure that all added content has been spellchecked.
+- [x] **Sensitive Information:** :no_entry: Before submitting, double-check to ensure no passwords, credentials, or local configurations are present in your changes.
+- [x] **Continuous Integration:** :green_circle: Confirm that the GitHub Actions CI build finishes successfully without test failures.
+
+## Database :floppy_disk:
+
+### Using Dev-Data for Local Development :construction_worker:
+For ease of development and to ensure consistency across local environments, 
+we provide a predefined dataset known as 'Dev-Data'. 
+This dataset is tailored to streamline the testing process and ensure that features, 
+including edge cases, are effectively covered.
+
+### Recommended Test User :bust_in_silhouette:
+To begin with, utilize the `test@example.com / test` user credentials. 
+This user has been populated with a comprehensive set of camps that should suffice for testing most features and scenarios.
+
+### Feedback on Dev-Data :loudspeaker:
+We constantly strive to improve our 'Dev-Data'. 
+If you identify gaps or believe there's an additional scenario it should cover, 
+please open an issue to let us know.
+
+### Documentation :mag:
+For a deeper understanding of 'Dev-Data', you can refer to its dedicated [README](./api/migrations/dev-data/README.md).
+
+### Consistent Testing Across Environments :globe_with_meridians:
+'Dev-Data' is replicated across all development environments. 
+We encourage its use for consistent testing. 
+When reporting an issue or bug, consider referencing a specific example from 'Dev-Data'. 
+Since the data, including IDs, remains consistent, it allows everyone to easily replicate and understand the behavior you're highlighting.
+
+## Discussions :speech_balloon:
+We understand that setting up a development environment can sometimes be tricky, 
+especially with varying systems and configurations.
+If you encounter any issues or face roadblocks during the setup, 
+please don't hesitate to open a Discussion on GitHub. 
+Our core team is happy to help you. 
+In fact, we encourage you to open a Discussion whenever you have a question. 
```

Comment:
```suggestion
In fact, we encourage you to open a [Discussion](./discussions) whenever you have a question. 
```

Reply:
Does this work? I tested it on my Branch and it did not, might be because thats on a fork. I added the URL to ensure this works locally and on forks.